ui/tests/integration/components/link-status-test.js

```diff
@@ -11,7 +11,7 @@ module('Integration | Component | link-status', function (hooks) {
 
   // this can be removed once feature is released for OSS
   hooks.beforeEach(function () {
-    this.owner.lookup('service:version').set('isEnterprise', true);
+    this.owner.lookup('service:version').set('version', '1.13.0+ent');
```

Comment:
will this continue to stay true after we increase the binaries? say when we hit the next 1.14 binary?

Reply:
I think the plan is to introduce this to OSS in the next version so it's likely only temporary. The problem with this particular line was that `isEnterprise` is a computed property and an error is thrown now when a computed property without a setter is overriden. Setting the version this way makes `isEnterprise` return `true`.